yoti-sdk-api/src/main/java/com/yoti/api/client/Anchor.java

```diff
@@ -10,23 +10,25 @@
 public interface Anchor {
 
     /**
-     * The type of Anchor - most likely either SOURCE or VERIFIER, but it's possible that new Anchor types will be added in future
-     *
+     * The type of Anchor - most likely "SOURCE" or "VERIFIER" but it's possible that new Anchor types will be added in future
      * @return String, naming the type of the anchor
      */
     String getType();
 
     /**
-     * For SOURCE Anchors, describes how the attribute value was obtained, for example this might be 'OCR' or 'NFC'
-     * For VERIFIER Anchors, describes the verification method used for the attribute
+     * <pre>
+     * For SOURCE Anchors, describes how the attribute value was obtained, will be one of: ['USER_PROVIDED', 'PASSPORT', 'DRIVING_LICENSE', 'PASSCARD', 'NATIONAL_ID']
      *
+     * For VERIFIER Anchors, describes the verification method used for the attribute, will be one of: ['YOTI_ADMIN', 'YOTI_IDENTITY', 'YOTI_OTP', 'PASSPORT_NFC_SIGNATURE', 'ISSUING_AUTHORITY', 'ISSUING_AUTHORITY_PKI']
+     * </pre>
      * @return String, further detailing how the attribute value was obtained
      */
     String getSubType();
 
     /**
      * Identifies the provider that either sourced or verified the attribute value.
-     * The range of possible values is not limited.  For a SOURCE anchor, expect values like PASSPORT, DRIVING_LICENSE...
+     * The range of possible values is not limited.
+     * For example, for a passport, this would either be "NFC" or "OCR"
```

Comment:
I also think these are the wrong way round - `For example, for a passport, this would either be "NFC" or "OCR"` is describing getSubType, and the range of possible Source/Verifier options listed above should be part of this `getValue` method 👍